node/impl/full/eth.go

```diff
@@ -369,10 +366,18 @@ func (a *EthModule) EthGetTransactionByHashLimited(ctx context.Context, txHash *
 	if txHash == nil {
 		return nil, nil
 	}
+	if a.ChainIndexer == nil {
+		return nil, errors.New("chain indexer is disabled; please enable the ChainIndexer to use the ETH RPC API")
```

Comment:
hey, maybe we should just make this error a reusable variable and then use it in the 3 places we do `ChainIndexer == nil`

Reply:
Done.